crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -176,6 +176,27 @@ def create_proposal
     base
   end
 
+  def disable_non_core_proposals
```

Comment:
As we have the architecture now with CrowbarService overriding/extending the ServiceObject I think it makes sense here.
Not sure if we manipulate much on NodeObjects in the lib either.